scripts/generate_cpp_dependency_json.py

```
@@ -0,0 +1,205 @@
+
+import argparse
+import json
+import logging
+import pathlib
+import time
+import urllib.parse
+import urllib.request
+
+logger = logging.getLogger(__name__)
+
+TOKEN = "glpat-EtaLqcDLtx9iZW6szKoN3GM6MQpvOjEKdTpoZmxxaQ8.01.170akjmvp"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "TOKEN"

(S105)